crm_lead_lost_reason/README.md

```diff
@@ -0,0 +1,23 @@
+Opportunity Lost Reason
```

Comment:
No, not really, they are read, but not correctly parsed. They are parsed as RestructuredText (RST). So better to put .rst so GitHub parses OK also.